README.md

```diff
@@ -23,18 +23,20 @@ In order to properly install the game, you'll have to follow these steps precise
   ux0:tai/fd_fix.skprx
 ```
 
+**Note** Don't install fd_fix.skprx if you're using repatch plugin
+
 - **Optional**: Install [PSVshell](https://github.com/Electry/PSVshell/releases) to overclock your device to 500Mhz.
 - Install `libshacccg.suprx`, if you don't have it already, by following [this guide](https://samilops2.gitbook.io/vita-troubleshooting-guide/shader-compiler/extract-libshacccg.suprx).
-- Obtain your copy of *Grand Theft Auto: San Andreas v2.00* legally (`com.rockstargames.gtasager` is not supported!) for Android in form of an `.apk` file and one or more `.obb` files (usually `main.8.com.rockstargames.gtasa.obb` and `patch.8.com.rockstargames.gtasa.obb` located in `/sdcard/android/obb/com.rockstargames.gtasa`). [You can get all the required files directly from your phone](https://stackoverflow.com/questions/11012976/how-do-i-get-the-apk-of-an-installed-app-without-root-access) and they can be extracted with whatever Zip extractor you prefer (eg: WinZip, WinRar, etc...).
-- Extract the `assets` folder from your `.apk` file to `ux0:data` and rename it to `gtasa`.
-- Extract the file `libGTASA.so` from the `lib/armeabi-v7a` folder inside your `.apk` file to `ux0:data/gtasa`.
-- Extract the the contents of `main.8.com.rockstargames.gtasa.obb` file to `ux0:data/gtasa`.
-- Extract the the contents of `patch.8.com.rockstargames.gtasa.obb` file to `ux0:data/gtasa`.
-- Extract [gamefiles.zip](https://github.com/TheOfficialFloW/gtasa_vita/releases/download/v1.0/gamefiles.zip) to `ux0:data/gtasa`.
-- **Optional**: For a more authentic console experience, copy the file `ux0:data/gtasa/data/360Default1280x720.cfg` to `ux0:data/gtasa/Adjustable.cfg`. This file is a leftover from the Xbox 360 version and provides you the console HUD (e.g. radar on bottom left).
+- Obtain your copy of *Grand Theft Auto: San Andreas v2.00* legally (`com.rockstargames.gtasager` is not supported!) for Android in form of an `.apk` file and one or more `.obb` files (usually `main.8.com.rockstargames.gtasa.obb` and `patch.8.com.rockstargames.gtasa.obb` located inside the `/sdcard/android/obb/com.rockstargames.gtasa/`) folder. [You can get all the required files directly from your phone](https://stackoverflow.com/questions/11012976/how-do-i-get-the-apk-of-an-installed-app-without-root-access) or by using an apk extractor you can find in the play store.The apk can be extracted with whatever Zip extractor you prefer (eg: WinZip, WinRar, etc...) since apk is basically a zip file. You can rename `.apk` to `.zip` to open them with your default zip extractor.
```

Comment:
` find in the play store.The apk can` add space.

Reply:
sure will do